### PR TITLE
Kjcho/add kafka input

### PR DESCRIPTION
**Pull request by kjcho-msft**
add eventhub for kafka input support

### CODE REVIEW

File: Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js [hunk 1]
```diff
@@ -124,6 +125,11 @@ export const inputTypes = [
         name: 'IoT Hub',
         disabled: false
     },
+    {
+        key: inputTypeEnum.eventforkafka,
```
Comment (barryt2) [inline Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js]: Can we order the name alphabetically?

Comment (kjcho-msft) [reply Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js]: changed to Kafka (Event Hub)

File: Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js [hunk 2]
```diff
@@ -124,6 +125,11 @@ export const inputTypes = [
         name: 'IoT Hub',
         disabled: false
     },
+    {
+        key: inputTypeEnum.kafkaeventhub,
+        name: 'Kafka (Event Hub)',
```
[the remark this answers is not in the document]
Comment (barryt2) [reply Website/Packages/datax-pipeline/src/modules/flowDefinition/flowModels.js]: according to documentation, event hub has a space between event and hub. we should stay consistent with what is used in Azure documentation

File: Website/Packages/datax-pipeline/src/modules/flowDefinition/flowSelectors.js [hunk 3]
```diff
@@ -201,7 +201,7 @@ function validateInput(input) {
     validations.push(input && input.properties);
 
     if (input.mode === Models.inputModeEnum.streaming) {
-        if (input.type === Models.inputTypeEnum.events) {
+        if (input.type === Models.inputTypeEnum.events || input.type === Models.inputTypeEnum.eventforkafka) {
```
[the remark this answers is not in the document]
Comment (kjcho-msft) [reply Website/Packages/datax-pipeline/src/modules/flowDefinition/flowSelectors.js]: changed to KafkaEventHub

**Pull request merged**